.ci.yaml

```diff
@@ -4016,6 +4016,8 @@ targets:
       tags: >
         ["devicelab", "hostonly", "mac"]
       task_name: plugin_test_ios
+      # Retry for flakes caused by https://github.com/flutter/flutter/issues/151772
+      presubmit_max_attempts: "2"
```

Comment:
What about extending the timeout on line 4010?

Reply:
Ahh, ok yeah, that's the *step* timeout. It looks like we're using the default (of 30 minutes), and this would require a recipe change to make that configurable: https://flutter.googlesource.com/recipes/+/refs/heads/main/recipes/devicelab/devicelab_drone.py#172

This LGTM for now.

Reply:
I think it has to be `test_timeout_secs` under properties: https://flutter.googlesource.com/recipes/+/refs/heads/main/recipes/devicelab/devicelab_drone.py#76

Reply:
Added `test_timeout_secs: 3600` to make the test step use the full hour.

Reply:
ahh yeah, you're right (I read this recipe wrong, not the first or last time)

Reply:
still lgtm